tools/benchmark_tool/openvino/tools/benchmark/benchmark.py

```diff
@@ -127,7 +136,7 @@ def async_inference_only(self, infer_queue, progress_bar):
               (self.duration_seconds and exec_time < self.duration_seconds) or \
               (iteration % self.nireq):
             idle_id = infer_queue.get_idle_request_id()
-            if idle_id in in_fly:
+            if idle_id in in_fly:       # Is this check neccessary?
```

Comment:
Why do you think that this check is not neccessary?

Reply:
From my investigation, this check is always 'False'. in_fly set will contain the idle_id if the infer_queue returns the same value twice+, which never occurred even in async mode. This also caused the shapes latency to be always '0.00'.

Reply:
@PiotrKrzem 
1. in_fly should contain all requests ids which was in use. If you have, for example, 100 requests, but number of iterations was 10, you should get latencies only of 10. 
2. few lines below is applicable for requests which were sent but was not processed in loop